src/styles/index.scss

```diff
@@ -1,3 +1,243 @@
+$primary-color: #00acdc;
+$font-color: #060b35;
+$light-grey: #616070;
+$font-family: 'Roboto', sans-serif;
+
+:root {
+  --blue-color: #00acdc;
+  --black-color: #060b35;
+}
+
+html {
+  margin: 0;
+  max-width: 1024px;
+  font-family: $font-family;
+  font-size: 12px;
+  line-height: 14.06px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: end;
+  gap: 243px;
+  height: 60px;
+  background-color: #fff;
+  font-family: $font-family;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  padding-right: 50px;
+}
+
+.logo {
+  display: flex;
+  position: absolute;
+  top: 10px;
+  left: 50px;
+  margin: 0;
+}
+
+.nav__link-list {
+  display: flex;
+  gap: 20px;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: $font-color;
+  cursor: pointer;
+  text-transform: uppercase;
+  position: relative;
+  transition: color 0.3s ease;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00bfff;
+}
+
+.is-active {
+  color: #00bfff;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: -19;
```

Comment:
The value for 'bottom' should include a unit (e.g., 'px'). Without a unit, this might not work as expected in some browsers.